measures/2018/measures-schema.yaml

```diff
@@ -1,7 +1,8 @@
-id: https://github.com/CMSgov/qpp-measures-data/versions/0.0.1/measures-schema.yaml
-$schema: http://json-schema.org/schema#
+$id: https://github.com/CMSgov/qpp-measures-data/versions/0.0.1/measures-schema.yaml
```

Comment:
Do we want to update this url? This url doesn't exist, and I don't know if it ever did. I know we've updated this schema many times and I'm now updating it again soo... @marimiyachi @kencheeto

Reply:
Hmm I don't have much context, but could we just delete it...?

Reply:
Hm ok. I'll just update it with `https://github.com/CMSgov/qpp-measures-data/blob/master/measures/2018/measures-schema.yaml` since it's apparently a unique identifier and a base url for $ref (though all of our ref urls are relative...) https://spacetelescope.github.io/understanding-json-schema/structuring.html#the-id-property